### ISSUE TITLE
[oc] doesn\t work as expected

### ISSUE THREAD

**Issue by eyenx** [whalebrew/whalebrew-packages#87]
```
/ # /usr/local/bin/oc
/bin/sh: /usr/local/bin/oc: not found
/ # /usr/local/bin/kubectl
/bin/sh: /usr/local/bin/kubectl: not found
```

**Comment by eyenx** [comment]
Assign me please. I'm gonna look into it

**Comment by tjamet** [comment]
it is probably because of alpine.
I understand that `https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz` provides a binary version, probably built on glibc linux.
if we move the base OS to `debian:stable-slim` we should probably solve our problems

**Comment by eyenx** [comment]
Yeah, I thought that too, but then I asked myself how `whalebrew/kubectl` solves that. It is also using alpine.

**Comment by tjamet** [comment]
true, maybe `kubectl` is not built the same way in both cases?

**Comment by eyenx** [comment]
Ill try with `debian:stable-slim`

**Comment by tjamet** [comment]
Hi!
Has this been closed by #88 ?

**Comment by eyenx** [comment]
Yes
